Fix block explorer guides (#386)
matter-labs/zksync-web-era-docs: docs/api/tools/block-explorer/block-view.md

# Browse blocks

![Browse blocks!](../../../assets/images/blocks.png "View blocks on zkSync")

> The block explorer home screen shows the 10 most recent blocks by default. To see a more complete overview of recent blocks, you can view all blocks. You can also click on a specific block number to see details about that particular block.
> You can access the blocks page from the top menu as well.

Reviewer: ```suggestion
- The block explorer home screen shows the 10 most recent blocks by default. 
To see a more complete overview of recent blocks, you can view all blocks. You can also click on a specific block number to see details about that particular block.
You can access the blocks page from the top menu as well.
```

Reviewer: Link to block explorer?

This sentence **- You can also access the blocks page from the top menu.** should be in the previous section as an introduction to the pic and should be something like:

**Access the blocks page from the top menu in the explorer dashboard**

Then you can go into the specifics.

![Access the block page](../../../assets/images/block-menu.png "Blocks menu")

> ## See all blocks

Reviewer: ```suggestion
## View all blocks
```

To see details of a specific block, just click on its block number. If you want to view more blocks, go to the [Blocks](https://explorer.zksync.io/blocks/) section in the navigation menu and use pagination to navigate to the next page.

> If you are not on the homepage, you can still search for a block by typing in its block number. A block has two main sections.

Reviewer: ```suggestion
Search for a specific block by typing in its block number.
```

Reviewer: Add pic.

![Browse all blocks](../../../assets/images/view-block.png "Browse all blocks")

A block has two main parts:

1. Block Header: It provides a summary of the block's information, along with a link to the previous block in the chain.

> 2. Transactions: It lists all the transactions that were included in the block.

Reviewer: ```suggestion
2. Transactions section containing all the transactions managed by the block.
```

![Single block page!](../../../assets/images/single-block.png "View a single block")

> ### Block headers

Reviewer: ```suggestion
### Block header information
```

> This table will help you understand what you'll see in a block. To learn more about blocks, check out the documentation on [Blocks](../../../dev/developer-guides/transactions/blocks.md).

Reviewer: A link to the same page was unnecessary.

> This table will help you understand what you'll see in a block. To learn more about blocks, check out the documentation on [Blocks](../../../dev/developer-guides/transactions/blocks.md).

Reviewer: Actually, you don't need this table. It's duplicate. Just link to the table in the docs instead.

| Value        | Description                                                                |
| ------------ | -------------------------------------------------------------------------- |
| Block number | The number of the current block. `null` if pending. [More info on block.number](../../../dev/developer-guides/transactions/blocks.md#block-number-and-timestamp-considerations)    |
| Block size   | The size of the block.                                                     |
| Timestamp    | The UNIX timestamp for when the block was collated. [More info on block.timestamp](../../../dev/developer-guides/transactions/blocks.md#block-number-and-timestamp-considerations)|
| Root hash    | The Cryptographic hash of the block header.                                |
| Status       | The status of the block, could be `sealed`, `included`, `finalized`, or `unfinalized`. |

### Transactions

> In this part, you can find a list of all the transactions included in this block. Transactions are processed in the order they are received, but we plan to introduce a "priority heap" in the future to sort transactions differently.

Reviewer: ```suggestion
In this section, you can find a list of all the transactions included in the block. Transactions are processed in the order they are received.
```

| Timestamp         | The block generation time in seconds since the Unix epoch.                                                                                               |
| Hash              | The hash of the transaction serving as the transaction’s ID.                                                                                             |
| From              | The account or smart contract, sending the transaction.                                                                                                  |
| To                | The account or smart contract, the transaction targeted.                                                                                                 |
| Fee               | The rate of fees associated with this transaction processing.                                                                                            |
| Tokens transferred| Details about the tokens(both `to` and `from` addresses) involved in the transactions, it includes, details like the assets, balance in the wallet address, and token address. |

> | Contract address  | The rate of fees collected by the network.                                                                                                               |

Reviewer: These aren't what I'm seeing on the block explorer?
